### PR TITLE
Add editor and fix styles

### PR DESCRIPTION
**Pull request by sumboid**

### CODE REVIEW

**Review by tpashkin — CHANGES_REQUESTED** [review]
Let's avoid vertical alignment. I agree it's convenient, but every time you add misaligned line, you have to realign everything. Diffs also look cleaner this way.

File: src/mug/components/page/styles.clj [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
    [:.page-row {:display "table-row"
                 :width   "100%"}]
+   [:.page-row.header {:height "40px"}]
    [:.page-cell {:display "table-cell"
                  :width   "100%"}]
-   [:.page-content {:background-color "red"
-                    :max-height "100%"
-                    :overflow-y "auto"}]])
+   [:.page-content {:height  "100%"
+                    :padding "15px"}]])                     ; TODO: get rid of this padding
```
Comment (tpashkin) [inline src/mug/components/page/styles.clj]: Any reason for more than one space before `;`?

File: src/mug/components/playback/controls/controls.cljs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+(defn- get-button [handle-start handle-stop state]
+  (if (= state :started) [stop-button handle-stop] [play-button handle-start]))
+
+(defn playback-controls [handle-start handle-stop]
+  (let [state (r/atom :stopped)]
+    (fn []
+      [:div.playback-controls
+       [:div.playback-button (get-button #(do
+                                            (handle-start)
```
Comment (tpashkin) [inline src/mug/components/playback/controls/controls.cljs]: Now that's just ugly

File: src/mug/styles/colors.clj [hunk 3]
```diff
@@ -0,0 +1,26 @@
+(ns mug.styles.colors)
+
+(def default-bg "#f9f5d7")
```
Comment (tpashkin) [inline src/mug/styles/colors.clj:3]: Use maps for this kind of thing. I intend to tweak coloring a bit, so you can leave this part in PR intact, just a tip

File: src/mug/components/buttons/icon/styles.clj [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+
+(def defaults
+  [[:.icon-button {:display "flex"
+                   :justify-content "center"
+                   :align-items "center"
+                   :width "40px"
+                   :height "40px"
+                   :border-radius "10px"
+
```
Comment (tpashkin) [inline src/mug/components/buttons/icon/styles.clj]: Yeah, this newline seems crucial

File: src/mug/components/buttons/icon/styles.clj [hunk 5]
```diff
@@ -0,0 +1,15 @@
+(ns mug.components.buttons.icon.styles
+  (:require [mug.styles.colors :as c]))
+
+
```
Comment (tpashkin) [inline src/mug/components/buttons/icon/styles.clj]: This one too

File: src/mug/components/page/styles.clj (hunk 1 above)
Comment (sumboid) [reply src/mug/components/page/styles.clj]: Done

File: src/mug/components/playback/controls/controls.cljs (hunk 2 above)
Comment (sumboid) [reply src/mug/components/playback/controls/controls.cljs]: Done

File: src/mug/components/buttons/icon/styles.clj (hunk 4 above)
Comment (sumboid) [reply src/mug/components/buttons/icon/styles.clj]: Done

File: src/mug/components/buttons/icon/styles.clj (hunk 5 above)
Comment (sumboid) [reply src/mug/components/buttons/icon/styles.clj]: Done

**Review by tpashkin — APPROVED** [review]

**Pull request merged**